anomalib/utils/callbacks/__init__.py

```diff
@@ -92,7 +92,7 @@ def get_callbacks(config: Union[ListConfig, DictConfig]) -> List[Callback]:
             nncf_config = yaml.safe_load(OmegaConf.to_yaml(config.optimization.nncf))
             callbacks.append(
                 nncf_callback(
-                    config=nncf_config,
+                    nncf_config=nncf_config,
```

Comment:
Since this is nncf_callback, `config` would indicate the `nncf_config`, wouldn't it? In this case, not sure if we need this renaming here.